packages/components/src/Components/SkeletonTable/SkeletonTable.js

```diff
@@ -47,12 +56,16 @@ SkeletonTable.propTypes = {
     sortBy: PropTypes.shape({
         index: PropTypes.number,
         direction: PropTypes.oneOf([ 'asc', 'desc' ])
-    })
+    }),
+    onSelect: PropTypes.bool,
```

Comment:
This should be `func`
```suggestion
    onSelect: PropTypes.func,
```